src/new_index/mempool.rs

```diff
@@ -8,13 +8,14 @@ use elements::{encode::serialize, AssetId};
 
 use std::collections::{BTreeSet, HashMap, HashSet};
 use std::iter::FromIterator;
-use std::sync::Arc;
+use std::sync::{Arc, RwLock};
 use std::time::{Duration, Instant};
 
 use crate::chain::{deserialize, Network, OutPoint, Transaction, TxOut, Txid};
 use crate::config::Config;
 use crate::daemon::Daemon;
 use crate::errors::*;
+use crate::error_chain::ChainedError;
```

Comment:
This is no longer necessary and causes an `unused import` warning

Reply:
Fixed.